<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculatrice à Notation Polonaise Inversée - Amhed &amp; JB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Pile 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Pile 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListView" name="listView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Complexe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Reel</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Rationnel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Entier</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Basique</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_25">
          <property name="text">
           <string>7</string>
          </property>
          <property name="shortcut">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_26">
          <property name="text">
           <string>8</string>
          </property>
          <property name="shortcut">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_27">
          <property name="text">
           <string>9</string>
          </property>
          <property name="shortcut">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_28">
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_29">
          <property name="text">
           <string>4</string>
          </property>
          <property name="shortcut">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_30">
          <property name="text">
           <string>5</string>
          </property>
          <property name="shortcut">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_31">
          <property name="text">
           <string>6</string>
          </property>
          <property name="shortcut">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButton_32">
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_33">
          <property name="text">
           <string>1</string>
          </property>
          <property name="shortcut">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_34">
          <property name="text">
           <string>2</string>
          </property>
          <property name="shortcut">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_35">
          <property name="text">
           <string>3</string>
          </property>
          <property name="shortcut">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_36">
          <property name="text">
           <string>*</string>
          </property>
          <property name="shortcut">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_37">
          <property name="text">
           <string>0</string>
          </property>
          <property name="shortcut">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_38">
          <property name="text">
           <string>.</string>
          </property>
          <property name="shortcut">
           <string>., ,</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_39">
          <property name="text">
           <string>space</string>
          </property>
          <property name="shortcut">
           <string>Space</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_40">
          <property name="text">
           <string>/</string>
          </property>
          <property name="shortcut">
           <string>/</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Divers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="toolTip">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Dépile X puis Y et effectue Y puissance X&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;X: Entier, Rationnel, Réel&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Y: Entier, Rationnel, Réel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>POW</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>INV</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>MOD</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>SQRT</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>SIGN</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>FACT (!)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>SQR</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>LN</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>CUBE</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>LOG</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>EVAL</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Trigo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_65">
          <property name="text">
           <string>SIN</string>
          </property>
          <property name="shortcut">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>Radian</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_66">
          <property name="text">
           <string>COS</string>
          </property>
          <property name="shortcut">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioButton_6">
          <property name="text">
           <string>Degré</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_67">
          <property name="text">
           <string>TAN</string>
          </property>
          <property name="shortcut">
           <string>T</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>Hyperbolic</string>
          </property>
          <property name="shortcut">
           <string>H</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Pile</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>SWAP</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_68">
          <property name="text">
           <string>CLEAR</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Del</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_72">
          <property name="text">
           <string>SUM</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_69">
          <property name="text">
           <string>DUP</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_71">
          <property name="text">
           <string>MEAN</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_70">
          <property name="text">
           <string>DROP</string>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSinus">
   <property name="text">
    <string>Sinus</string>
   </property>
  </action>
  <action name="actionCos">
   <property name="text">
    <string>Cosinus</string>
   </property>
  </action>
  <action name="actionTan">
   <property name="text">
    <string>Tangente</string>
   </property>
  </action>
  <action name="actionSinus_2">
   <property name="text">
    <string>Sinus</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
